charts/hedera-mirror-common/values.yaml

```
@@ -145,8 +145,17 @@ prometheus-adapter:
     requests:
       cpu: 25m
       memory: 50Mi
-  rules:
-    default: false
+  rules:  # This is a custom rule which exposes (requests_per_second) from the prometheus adapter to be used by any Horizontal Pod Autoscaler.
+    custom:
+      - seriesQuery: 'api_all_request_total{namespace!="",pod!=""}'
+        resources:
+          overrides:
+            namespace: { resource: "namespace" }
+            pod: {resource: "pod"}
+        name:
+          matches: "^(.*)_all_request_total$"
+          as: "requests_per_second"
+        metricsQuery: "sum(rate(<<.Series>>{<<.LabelMatchers>>}[1m])) by (<<.GroupBy>>)"
```

I've found that 1m intervals are a bit costly to calculate and unreliable.
```suggestion
        metricsQuery: "sum(rate(<<.Series>>{<<.LabelMatchers>>}[3m])) by (<<.GroupBy>>)"
```